src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -783,6 +784,37 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
         return ResponseEntity.ok(stats);
     }
 
+    /**
+     * GET /courses/stats-for-management-overview :
+     *
+     * @param courseIds course Ids to get the stats for
+     * @param periodIndex period index for the user activity stats
+     * @return ResponseEntity with status
+     */
+    @GetMapping(value = "/courses/stats-for-management-overview")
```

Comment:
```suggestion
    @GetMapping("/courses/stats-for-management-overview")
```
Is it necessary to use `value = ` here?